src/components/timeline/Timeline.svelte

```diff
@@ -146,6 +148,31 @@
     setRowsMaxHeight(timelineDiv, xAxisDiv, timelineHistogramDiv);
   });
 
+  onDestroy(() => {
+    removeDPRChangeListener();
```

Comment:
The e2e tests seem to be failing because of this line, but only in the case when loading a plan directly from the URL (e.g. going to `localhost/plans/1`). In this case onDestroy is called before the `removeDPRChangeListener` is set.

```suggestion
  onDestroy(() => {
    if (removeDPRChangeListener) {
      removeDPRChangeListener();
    }
  });
```

Reply:
Hm but then why does the conditional check succeed?

Reply:
Good catch, added a check.